src/networktables/NetworkTable.cpp

```diff
@@ -73,6 +73,14 @@ void NetworkTable::SetTeam(int team) {
     servers[3] = std::make_pair(oss.str(), s_port);
   }
 
+  // roboRIO-<team>-FRC.frc-field.local
+  llvm::SmallString<32> field_local;
```

Comment:
This should be bigger than 32 otherwise you end up with an allocation for many team numbers.